.prettierignore

```diff
@@ -5,6 +5,7 @@ canon-docs/.next
 canon-docs/public
 canon-docs/out
 packages/canon/css
```

Comment:
Should you remove this line?

Reply:
Ah yep with the copy setup this can be removed 👍